go.mod

```diff
@@ -10,10 +10,10 @@ require (
 	golang.org/x/oauth2 v0.25.0 // indirect
 	golang.org/x/term v0.28.0 // indirect
```

Comment:
I think it is strange that these 3 indirect requires are not in the block below. Maybe we can take the chance to fix this too.

Reply:
It's doesn't match the issue, We can fix it individually